rtCamp/web-stories-wp
# Feature/web stories block styles

## Summary

- Replaces 'Show Cover image' control with 'Show Story Player' control to embed stories in-place.
- Adds new control to make story cards have rounded corners.

## Relevant Technical Choices

- By default, the blocks will show cover images.
- The control to toggle the corner style will be available only when cards are showing the cover image.
 
## To-do

- Maybe come up with a different name for the 'Replace cover image with story player' control.

## Testing Instructions

- Select 'Latest Web Stories' or 'Selected Block Stories' block in Editor.
- Observe new controls on the righthand sidebar.
- Toggle 'Replace Cover image with story player' to embed stories in-place.

![Screenshot 2020-11-23 at 4 43 48 PM](https://user-images.githubusercontent.com/6906779/99955914-20ed1d00-2dab-11eb-87f7-e79895447f15.png)

- Toggle 'Show square corners' to make card corners rounded.

---



Fixes #38
